igvm/hypervisor.py

```diff
@@ -200,6 +201,20 @@ def check_vm(self, vm, offline):
                 .format(self.dataset_obj['num_cpu'], vm.dataset_obj['num_cpu'])
             )
 
+        # CPU usage of HV below threshold?
+        if vm.hypervisor:
+            hv_cpu_threshold = HYPERVISOR_CPU_THRESHOLDS[
+                self.dataset_obj['hardware_model']]
+            hv_cpu_util_overall = self.hv_cpu_util_overall(vm)
+
+            if hv_cpu_util_overall > hv_cpu_threshold:
+                raise HypervisorError(
```

Comment:
This part of code is for incompatibility of a HV and a VM. It means that it might be impossible to create new VMs at all. And it won't work nicely with scenarios with HVs with low, occasional load. Maybe it is enough to use this information only for sorting HVs, not to fully exclude them?

Reply:
Sorting will take place in the preferences already. This shouldn't break initial VM setup, because `vm.hypervisor` is not set.

Reply:
I agree with @TuxPowered42 This should be part of finding a suitable hypervisor in the preferences. The checks in here should just be a safety need to avoid trying to do something that will fail for sure. Creating/Migrating a VM to a loaded hypervisor is probably a bad idea but should not be denied.

Reply:
I removed this part of code, so only the Preference is left.